### PR TITLE
Create search

### PR DESCRIPTION
**Pull request by zachjamesgreen**
Created search component and search results component . tests for those components are passing. added query string package to parse the query from the path since react doesn't do it for us anymore. created header component with logo and search component and rendered in it.

## Pull request checklist
Please check if your PR fulfills the following requirements:
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Rubocop has passed locally and any fixes were made for failures
- [ ] Build is passing
    *build is not passing because AppActions.js is deleted*
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):
## What is the current behavior?

Issue Number: 
closes #
## What is the new behavior?

- 
- 
- 
## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by twarbelow** [comment]
does it matter that ci isn't passing yet?

**Comment by zachjamesgreen** [comment]
yeah a failing build is fin for now. there isn't a config file for circle to find so it fails it

File: src/components/SearchResults.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+export default class SearchResults extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {query: this.parseParams(props)}
+  }
+
+  parseParams = (props) => {
+    const { location: { search } } = props;
+    let parsed = queryString.parse(search);
+    return parsed.q;
```
Comment (markcyen) [inline src/components/SearchResults.js:14]: Sweet search box...

File: src/components/SearchResults.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    let image = 'https://via.placeholder.com/200'
+    return(
+      <div className="font-playfair font-normal text-3xl text-center mx-56">
+        <h2 className="mb-16">Search results for {this.state.query}</h2>
+        <div className="grid grid-cols-5 gap-4 justify-items-stretch">
+          {array.map((i) => {
+            return (
+              <div className="justify-self-center" key={i} id={i}>
+                <img src={image} alt="" />
+                <a className="text-base mt-4 hover:underline cursor-pointer">Cocktail {i}</a>
```
Comment (markcyen) [inline src/components/SearchResults.js:30]: Looks like when a user hovers over this image, it'll show something? Nice feature if that is the case...

File: src/test/Search.test.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import Search from '../components/Search.js';
+import { render, unmountComponentAtNode  } from 'react-dom';
+import { act } from "react-dom/test-utils";
+
+let container = null;
+beforeEach(() => {
+  // setup a DOM element as a render target
```
Comment (markcyen) [inline src/test/Search.test.js:8]: Oh... I'll want to learn about testing in JS.

**Review by markcyen — APPROVED** [review]
Looks great to me! Can't wait to see it!

**Review by twarbelow — CHANGES_REQUESTED** [review]
- there are file naming descrepencies that should be addressed, but that isn't too big a deal
- the main thing I'm having an issue with is 'SearchResults' being undefined in the app.js. I've attached a screenshot
Screen Shot 2021-07-30 at 12 48 16 PM

**Review by twarbelow — APPROVED** [review]

**Pull request merged**